content/plugins/openstack.md

```diff
@@ -108,14 +108,14 @@ custom_configuration:
 
 **Properties:**
 
-  * `server` Key-value server configuration as described in the [OpenStack compute create server API](http://developer.openstack.org/api-ref-compute-v2.html#compute_servers). (**Deprecated** - Use the `args` input in create operation instead.)
+  * `server` Key-value server configuration as described in the [OpenStack compute create server API](http://developer.openstack.org/api-ref-compute-v2.html#compute_servers). app_svc
```

Comment:
What's app_svc and why is it hanging out at the end of the line? (Same for all others below)